lib/exhal/navigation.ex

```diff
@@ -1,8 +1,8 @@
 defmodule ExHal.Navigation do
-  @client_module Application.get_env(:exhal, :client, ExHal.Client)
+  # client_module() Application.get_env(:exhal, :client, ExHal.Client)
```

Comment:
Probably want to take this out if it's unused

Reply:
correct. Consider it gone.